consensus/bor/bor.go

```diff
@@ -386,11 +386,6 @@ func (c *Bor) verifyHeader(chain consensus.ChainHeaderReader, header *types.Head
 		return fmt.Errorf("invalid gasLimit: have %v, max %v", header.GasLimit, gasCap)
 	}
 
-	// If all checks passed, validate any special fields for hard forks
-	if err := misc.VerifyForkHashes(chain.Config(), header, false); err != nil {
```

Comment:
@0xsharma please check this

Reply:
Yes, this function just compared the EIP150 blockhash and has been removed in the later versions of geth.